### PR TITLE
Add roadmap to README.md

### PR DESCRIPTION
**Pull request by woop**
Moves the roadmap to our README.md and restructures it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,11 +123,81 @@ pprint(feature_vector)
 }
 ```
 
+## Roadmap
```
Comment (felixwang9817) [inline README.md:126]: Any particular reason you're moving the roadmap to Github? I don't mind, just wondering why.

File: README.md [hunk 2]
```diff
@@ -123,11 +123,81 @@ pprint(feature_vector)
...
+    - [ ]  Data profiling and validation (Great Expectations)
+    - [ ]  Training-serving skew detection
+    - [ ]  Drift detection
+    - [ ]  Alerting
+
+- [ ]  **Feature Discovery and Governance**
+    - [x]  Python SDK for creating and browsing objects
+    - [x]  CLI for browsing objects
+    - [x]  Model-centric tracking (feature services)
+    - [ ]  Feast Web UI
```
Comment (felixwang9817) [inline README.md:193]: A few thoughts on adding more details. First, we could put `(Scheduled for development in next N months)` next to the items we plan on tackling soon. Second, we could add a blurb down below telling folks that we welcome contributions and would be happy to help folks with building one of these features. Third, we could add some way for folks to let us know what they think about our prioritization (e.g. maybe some simple Google form asking them to rank their prioritization of the roadmap features, or voice their feature requests, etc.).

Comment (woop) [reply README.md:193]: > Third, we could add some way for folks to let us know what they think about our prioritization

I think our prioritization then needs to be clear. Would it consist only out of `Scheduled for development in next N months`? I think we should identify the stages

* Planned for Q1/2/3/4
* In design phase (see RFC-1234)
* In development
* Contribution welcome

Comment (felixwang9817) [reply README.md:193]: Those stages look good to me.

**Review by felixwang9817 — APPROVED** [review]
/lgtm

**Pull request merged**